pages/common/bosh.md

```diff
@@ -0,0 +1,35 @@
+# bosh
+
+> Cli tool to deploy and manage the bosh director.
+
+- Deploy the director:
+
+`bosh create-env {{bosh.yml}} --state={{state.json}} --vars-store={{creds.yml}} -v {{custparam}}`
+
+- Create local alias for director:
+
+`bosh alias-env {{name}} -e {{192.168.1.123}} --ca-cert {{$(bosh int ./creds.yml --path /director_ssl/ca)}}`
+
+- Login to the director:
+
+`bosh login -e {{environment}} --client={{username}} --client-secret={{password}}`
+
+- List environments:
+
+`bosh environments`
+
+- List environment vm's:
```

Comment:
"environment vm's" ? Did you mean to say "environment's vms" ?

Reply:
no i meant environment virtual machines. i will change it to reflect this.